manifests/resource/mailhost.pp

```diff
@@ -210,17 +266,46 @@
   if $ssl_port == undef or $listen_port != $ssl_port {
     concat::fragment { "${name}-header":
       target  => $config_file,
-      content => template('nginx/mailhost/mailhost.erb'),
       order   => '001',
+      content => epp('nginx/mailhost/mailhost.epp', {
+          has_ipaddress6        => $has_ipaddress6,
+          ipv6_enable           => $ipv6_enable,
+          ipv6_listen_ip        => $ipv6_listen_ip,
```

Comment:
If only this would be that easy :)
```
➜  puppet apply -e '$a = "1.2.3.4"; notice(Array($a))'
Notice: Scope(Class[main]): [1, ., 2, ., 3, ., 4]
```

Reply:
Aha! I found!
```
➜  puppet apply -e '$a = "1.2.3.4"; $b = ["1.2.3.4", "1.1.1.1"]; notice(Array.new($a, true), Array.new($b, true))'
Notice: Scope(Class[main]): [1.2.3.4] [1.2.3.4, 1.1.1.1]
```